week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -55,8 +40,87 @@ let answer;
 // - positive
 // - negative
 // - very negative
-function checkAnswer() {}
-
+function checkAnswer(answer) {
+  let number = Math.ceil(Math.random() * 5);
+  // ## Very positive
+  // It is certain.
+  // It is decidedly so.
+  // Without a doubt.
+  //   Yes - definitely.
+  // You may rely on it.
+  if (answer === "Very positive") {
+    if (number === 1) {
+      return "It is certain.";
+    } else if (number === 2) {
+      return "It is decidedly so.";
+    } else if (number === 3) {
+      return "Without a doubt.";
+    } else if (number === 4) {
+      return "Yes - definitely.";
+    } else if (number === 5) {
+      return "You may rely on it.";
+    }
+    // ## Positive
+    // As I see it, yes.
+    // Most likely.
+    // Outlook good.
+    //   Yes.
+    // Signs point to yes.
+  }
+  if (answer === "Positive") {
+    if (number === 1) {
+      return "As I see it, yes.";
+    } else if (number === 2) {
+      return "Most likely.";
+    } else if (number === 3) {
+      return "Outlook good.";
+    } else if (number === 4) {
+      return "Yes.";
+    } else if (number === 5) {
+      return "Signs point to yes.";
+    }
+  }
+  // ## Negative
+  // Reply hazy, try again.
+  // Ask again later.
+  // Better not tell you now.
+  // Cannot predict now.
+  // Concentrate and ask again.
+  if (answer === "Negative") {
+    if (number === 1) {
+      return "Reply hazy, try again.";
+    } else if (number === 2) {
+      return "Ask again later.";
+    } else if (number === 3) {
+      return "Better not tell you now.";
+    } else if (number === 4) {
+      return "Cannot predict now.";
+    } else if (number === 5) {
+      return "Concentrate and ask again.";
+    }
+  }
+  // ## Very negative
+  // Don't count on it.
+  // My reply is no.
+  // My sources say no.
+  // Outlook not so good.
+  // Very doubtful.
+  if (answer === "Very negative") {
+    if (number === 1) {
+      return "Don't count on it.";
+    } else if (number === 2) {
+      return "My reply is no.";
+    } else if (number === 3) {
+      return "My sources say no.";
+    } else if (number === 4) {
+      return "Outlook not so good";
+    } else if (number === 5) {
+      return "Very doubtful";
+    }
+  }
+}
+// and return the answer.
+console.log(`level of posibility : ${answer}`);
```

Comment:
This code is right, but it's slightly in the wrong place. Remember how when we define a function, we don't run any of the code in it until someone calls it, but code outside of functions gets run straight away - so this code gets called _before_ anyone calls `shakeBall` (which is what assigns a variable to the `answer` variable), which means that `answer` isn't set here (which is why you'll see "level of posibility : undefined" logged).

If you move this code to happen in a function where we know `answer` has a value, you should see the right value get logged :)

Reply:
Hello,
Because of change in the master branch I had to make a new branch, name is: week 1 final changes. 
I fixed these mistakes there.
thank you for your comments.

Reply:
sorry I deleted that because of a mistake, I will upload it here shortly, but I want to ask a question and that is why I couldn't answer with or(||) tag rather than repeat one command over and over.

Reply:
it is a new pull request, named:
changes are done.